examples/create-react-app/public/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "React App",
+  "name": "Create React App Sample",
```

Comment:
Should we give this a better name, like `PDF.js rendering example for React`, and a short name?

Reply:
Done :)